daemons/pacemakerd/pcmkd_subdaemons.c

```diff
@@ -493,15 +493,17 @@ start_child(pcmk_child_t * child)
 
         if(gid) {
             // Whether we need root group access to talk to cluster layer
-            bool need_root_group = TRUE;
+            static bool need_root_group = true;
 
-            if (is_corosync_cluster()) {
+#if SUPPORT_COROSYNC
+            if (need_root_group && is_corosync_cluster()) {
```

Comment:
This is only a performance benefit for corosync clusters (which of course is the only kind we support at the moment). But if we had some other cluster layer that needed the root group, it would have to run is_corosync_cluster() every time. Probably not worth caring about now ...

Reply:
ah, true. May go ahead and make it global as part of whatever my next PR is